style.css

```diff
@@ -0,0 +1,80 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  z-index: 99999999;
+  background-color: #fff;
+}
+
+.welc {
+  color: #172b4f;
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+}
+
+.menu {
+  color: #172b4f;
+  font-size: 12px;
+}
```

Comment:
- kindly your content and list of icons haven't been centered as in the Figma design and your main title needs some padding-top and background image has been cut please stick to the Figma design to have better Ui :+1: 
### your design

![Screenshot from 2023-02-17 17-59-53](https://user-images.githubusercontent.com/52010789/219690300-9dc4e1d2-fa7a-4142-a568-248535ffb8a4.png)
### figma design
![Screenshot from 2023-02-17 17-59-12](https://user-images.githubusercontent.com/52010789/219690315-fbc7184e-a0e3-4263-8bb0-9a5b911fbc9e.png)